samples/data-grid/demo/filters/demo.css

```diff
@@ -0,0 +1,6 @@
+@import url("https://code.highcharts.com/dashboards/css/datagrid.css");
+
+#container {
+    height: 250px;
+    margin: 8px;
```

Comment:
Can we do something about the `8px` margin everywhere?

Reply:
@karolkolodziej what do you mean? Apply the style to all demos?

Reply:
Maybe the data grid container should have that margin by default?